lib/routes/sohu/mobile.ts

```diff
@@ -0,0 +1,109 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import cache from '@/utils/cache';
+import * as cheerio from 'cheerio';
+import logger from '@/utils/logger';
+
+export const route: Route = {
+    path: '/mobile',
+    categories: ['new-media', 'popular'],
+    example: '/sohu/mobile',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['m.sohu.com/limit'],
+            target: '/mobile',
+        },
+    ],
+    name: '首页新闻',
+    maintainers: ['asqwe1'],
+    handler,
+    description: '订阅手机搜狐网的首页新闻',
+};
+
+async function handler() {
+    try {
+        const response = await ofetch('https://m.sohu.com/limit', {
+            headers: {
+                'User-Agent': 'Mozilla/5.0 (iPhone; CPU iPhone OS 15_0 like Mac OS X)'
+            }
+        });
```

Comment:
```suggestion
        const response = await ofetch('https://m.sohu.com/limit');
```
The default UA of https://github.com/DIYgod/RSSHub/blob/f19a725157ff7ea4c0bdd04f7708ff387b2926d8/lib/config.ts#L458 works